lib/mongo_trails/mongo_support/version.rb

```diff
@@ -31,6 +31,8 @@ class Version
 
     increments :integer_id, scope: -> { PaperTrail::Version.prefix_map }
 
+    index({ item_type: -1, item_id: -1 })
```

Comment:
I think this is OK. this might be a built-in index, why not?
These kinds of selects are the most common. I like PR. However, if the prefix is dynamic, then this index will not work.